LEAF_Nexus/templates/site_elements/generic_dialog.tpl

```diff
@@ -3,6 +3,6 @@
         <div id="genericDialogbutton_cancelchange" style="display: none"></div>
         <div id="genericDialogbutton_save" style="display: none"></div>
         <div id="genericDialogloadIndicator" style="visibility: hidden; z-index: 9000; position: absolute; text-align: center; font-size: 24px; font-weight: bold; background-color: #f2f5f7; padding: 16px; height: 400px; width: 526px"><img src="images/largespinner.gif" alt="loading..." /></div>
-        <div id="genericDialogxhr" style="width: 540px; height: 420px; padding: 8px; overflow: auto; font-size: 12px"></div>
+        <div id="genericDialogxhr" style="min-width: 540px; min-height: 420px; width: 100%; height: 100%; padding: 8px; overflow: auto; font-size: 12px"></div>
```

Comment:
Having height at 100% is causing the dialog to expand to the whole screen and cuts off some content in form editor section.